src/index.ts

```diff
@@ -12,13 +13,15 @@ const defaultAxiosInstance = axios.create({
 
 const SQL = new SQLModule.SqlRestApiClient(defaultAxiosInstance);
 const Models = new ModelsModule.ModelsRestApiClient(SQL);
+const Projects = new ProjectsModule.ProjectsRestApiClient(defaultAxiosInstance);
 /**
  * Initializes the MindsDB SDK and authenticates the user if needed.
  * @param {ConnectionOptions} options - Options to use for initialization
  */
 const connect = async function (options: ConnectionOptions): Promise<void> {
   const httpClient = options.httpClient || defaultAxiosInstance;
   SQL.client = httpClient;
+  Projects.client = httpClient;
```

Comment:
In the case for the `Project` will be cool to use destructuring on properties, for example

`const { client, session } = Project;`

besides, that's will be more readeable

Reply:
I agree